frontend/components/Social.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box } from "@chakra-ui/react";
+import Link from "next/link";
+import React from "react";
+import Splash from "./Splash";
+
+export type SocialProps = React.PropsWithChildren<{
+  link: string;
```

Comment:
let's keep conventions and call this `href` (`url` would also be fine, but let's keep href)